orderapi/src/main/java/fi/hel/verkkokauppa/order/service/order/OrderService.java

```diff
@@ -159,23 +173,6 @@ public List<Order> findByUser(String userId) {
         return orderRepository.findByUser(userId);
     }
 
-    public Order findByNamespaceAndUser(String namespace, String user) {
```

Comment:
I noticed that there were a few methods here that are not used at all anywhere in the project, so I removed them.